src/components/Slider/BaseSlider/BaseSlider.tsx

```diff
@@ -13,7 +13,7 @@ import './BaseSlider.scss';
 const b = block('base-slider');
 
 type BaseSliderProps = {stateModifiers: StateModifiers} & Omit<
-    SliderProps,
+    SliderProps<number | [number, number]>,
```

Comment:
Why is this done here? this leads to the need to specify the @ts-expect-error directive at line 25

Reply:
Sorry for misunderstanding) 
I'd like to know why do we need this changes in types at all?)

Reply:
Changes give more correct types for `value` and `onUpdate`. Before this, type of value always `number | number[]` for `BaseSlider` and `number | [number | number]` for `Slider`. After this, if `value` has type 'number' then `onUpdate` also receives only values of type `number`. The same thing applies to the `[number, number]` type.

Reply:
ok, I see. thanks)